core/lib/multivm/src/versions/vm_latest/constants.rs

```diff
@@ -101,17 +113,17 @@ pub const VM_HOOK_POSITION: u32 = RESULT_SUCCESS_FIRST_SLOT - 1;
 pub const VM_HOOK_PARAMS_COUNT: u32 = 2;
 pub const VM_HOOK_PARAMS_START_POSITION: u32 = VM_HOOK_POSITION - VM_HOOK_PARAMS_COUNT;
 
-pub(crate) const MAX_MEM_SIZE_BYTES: u32 = 16777216; // 2^24
+pub(crate) const MAX_MEM_SIZE_BYTES: u32 = 24000000;
```

Comment:
mention why was it changed. And what's the relation to USED_BOOTLOADER_MEMORY_BYTES